File: src/main/java/xyz/lncvrt/galaxyboxpvp/commands/Sky.java
package xyz.lncvrt.galaxyboxpvp.commands;

import net.kyori.adventure.text.minimessage.MiniMessage;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.jetbrains.annotations.NotNull;

public class Sky implements CommandExecutor {
    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String @NotNull [] args) {
        if (sender instanceof Player player) {
            player.performCommand("warp sky");
            player.sendMessage(MiniMessage.miniMessage().deserialize("<green>You can do <b><u><click:suggest_command:'/skyrtp'>/skyrtp</click></u></b> (click the bold text) to randomly teleport around here!"));
        }
        return true;
    }
}

File: src/main/kotlin/xyz/lncvrt/galaxyboxpvp/commands/Sky.kt
package xyz.lncvrt.galaxyboxpvp.commands

import org.incendo.cloud.kotlin.MutableCommandBuilder
import org.sayandev.stickynote.bukkit.command.BukkitCommand
import org.sayandev.stickynote.bukkit.command.BukkitSender
import org.sayandev.stickynote.bukkit.extension.sendComponent

object Sky : BukkitCommand("sky") {
    override fun rootBuilder(builder: MutableCommandBuilder<BukkitSender>) {
        builder.handler { context ->
            val player = context.sender().player() ?: return@handler

            player.performCommand("warp sky")
            player.sendComponent("<green>You can do <b><u><click:suggest_command:'/skyrtp'>/skyrtp</click></u></b> (click the bold text) to randomly teleport around here!")
        }
    }
}
